hardhat.config.ts

```diff
@@ -117,7 +115,7 @@ const config: HardhatUserConfig = {
             chainId: 1,
             url: process.env.MAINNET_NODE || '',
             accounts: {
-                mnemonic: process.env.MNEMONIC || '',
+                mnemonic: process.env.PROD_MNEMONIC || '',
             },
         },
         ropsten: {
```

Comment:
If this is a cleaning story, can you please remove:
- ropsten
- rinkeby
- kovan
:pray:

Reply:
Yes [Clean networks](https://github.com/iExecBlockchainComputing/PoCo/pull/53/commits/6c55580f0096578ffb4aab4cd4c7541bc971b94f)

Reply:
Thanks!